### PR TITLE
Update sortDependencies to 0.9

### PR DESCRIPTION
**Pull request by ZacSweers**
https://github.com/square/gradle-dependencies-sorter/blob/main/CHANGELOG.md#version-09

### CODE REVIEW

File: platforms/intellij/skate/build.gradle.kts [hunk 1]
```diff
@@ -118,39 +117,20 @@ configurations
...
-      "org.jetbrains.kotlin",
-    )
-
-    pluginVerifier()
-    zipSigner()
-    instrumentationTools()
-
-    testFramework(TestFrameworkType.Platform)
-    testFramework(TestFrameworkType.Bundled)
-  }
```
Comment (ZacSweers) [inline platforms/intellij/skate/build.gradle.kts:152]: welp it broke all of this

File: platforms/intellij/skate/build.gradle.kts [hunk 2]
```diff
@@ -118,39 +117,20 @@ configurations
...
-  implementation(libs.okio)
   implementation(libs.kaml)
   implementation(libs.kotlinx.serialization.core)
   implementation(libs.okhttp)
   implementation(libs.okhttp.loggingInterceptor)
-  implementation(projects.platforms.intellij.compose, exclusions)
-  implementation(projects.tools.tracing, exclusions)
+  implementation(libs.okio)
 
-  intellijPlatform {
```
Comment (linhpha) [inline platforms/intellij/skate/build.gradle.kts:134]: why is this block removed?

Comment (ZacSweers) [reply platforms/intellij/skate/build.gradle.kts:134]: see below :). Looks like it's a bug

Comment (ZacSweers) [reply platforms/intellij/skate/build.gradle.kts:134]: [fbc1758](https://github.com/slackhq/foundry/pull/1027/commits/fbc1758069043f7571d9744c33b31510036cd1fb)

**Review by linhpha — APPROVED** [review]

**Review by jstewart5000 — APPROVED** [review]

**Comment by ZacSweers** [comment]
still seeing a couple bugs I wanna look into before merging

**Pull request closed without merging**